limbo/limbo.py

```diff
@@ -128,7 +130,11 @@ def handle_message(event, server):
         return handle_bot_message(event, server)
 
     try:
-        msguser = server.slack.server.users[event["user"]]
+        msguser = event["user"]
+        if msguser not in server.slack.server.users:
+            user = json.loads(server.slack.server.api_call("users.info", post_data={"msguser": msguser}))["user"]
```

Comment:
It's also why the tests fail; the slack.server mock doesn't have an `api_call` function. Rather than add it, I'd just move this to slackrtm.